### ISSUE TITLE
Add "security" mode

### ISSUE THREAD

**Issue by davidphayanouvong** [zekroTJA/shinpuru#360]
### Type

A new feature

### Description

It would be good to implement the a feature to protect the server from hack.

if you run a command manually, remove the `view permission` to everyone, remove all moderation role to everyone and create a read only channel with a custom message

**Comment by itsDkiller** [comment]
This would require Shinpuru to log all user specific roles, like Boosters, Verfied and even admin roles.

In addition to that: Shinpuru (and self-hosted instances would need to have an extremely large permission set on each guild.
Other bots could be brought to infunctionality, as they could be used to log things at certain events.

**Comment by voxain** [comment]
bump - Still personally think this is a good idea, since one of my recent suggestions [on discord (haven't created an issue for this yet )](https://discord.com/channels/1079207548042878976/1079207549200498780/1086005076780384377) would benefit from a "killswitch" like that aswell. 
Though it should be implemented with great care and inform users about the risks of giving the bot such an immense set of permissions.

**Comment by zekroTJA** [comment]
Yep, I would take this into the next roadmap (pretending there is a roadmap :D). Permission wise shinpuru already needs a lot of crucial permissions to operate. There is actually not a lot more left to ask for. But if you want to have a moderation bot with some amount of control over the server, sure it needs a lot of permissions.
